kratos/solving_strategies/schemes/residual_based_bdf2_displacement_scheme.h

```diff
@@ -0,0 +1,600 @@
+//    |  /           |
```

Comment:
suggestion: why don't you make it to also consider the BDF1 scheme? it is first order accurate but useful if you do want a lot of dissipation, for example in pseudo-dynamic problems

Reply:
I can do it